codexctl/device.py

```diff
@@ -53,12 +110,7 @@ def __init__(
             except FileNotFoundError:
                 machine_contents = "tests"
 
-        if "reMarkable Ferrari" in machine_contents:
-            self.hardware = "ferrari"
-        elif "reMarkable 1" in machine_contents:
-            self.hardware = "reMarkable1"
-        else:
-            self.hardware = "reMarkable2"
+        self.hardware = HardwareType.parse(machine_contents)

```

Comment:
This appears to have changed handling, previously self.hardware would not be set if the file was not found, now it's defaulting to the rM2.

Reply:
You are right, I seem to have misread that every single time as another elseif, whoops.